pkg/sharing/replicator.go

```diff
@@ -326,38 +326,19 @@ func (s *Sharing) callRevsDiff(inst *instance.Instance, m *Member, creds *Creden
 			"Authorization": "Bearer " + creds.AccessToken.AccessToken,
 		},
 		Body: bytes.NewReader(body),
-	})
+	}
+	var res *http.Response
+	res, err = request.Req(opts)
 	if err != nil {
 		return nil, err
 	}
 	if res.StatusCode/100 == 4 {
 		res.Body.Close()
-		if err = creds.Refresh(inst, s, m); err != nil {
-			return nil, err
-		}
-		res, err = request.Req(&request.Options{
-			Method: http.MethodPost,
-			Scheme: u.Scheme,
-			Domain: u.Host,
-			Path:   "/sharings/" + s.SID + "/_revs_diff",
-			Headers: request.Headers{
-				"Accept":        "application/json",
-				"Content-Type":  "application/json",
-				"Authorization": "Bearer " + creds.AccessToken.AccessToken,
-			},
-			Body: bytes.NewReader(body),
-		})
+		res, err = RefreshToken(inst, s, m, creds, opts)
```

Comment:
It won't work: the body is a stream, and it is consumed on the first request. When it is retried in `RefreshToken`, the body will be empty.